persistence/model/src/main/java/gov/va/vro/persistence/model/VeteranFlashIdEntity.java

```diff
@@ -0,0 +1,31 @@
+package gov.va.vro.persistence.model;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.GenericGenerator;
+
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "veteran_flash_id")
+public class VeteranFlashIdEntity {
+
+  @Id
+  @GeneratedValue(generator = "uuid")
+  @GenericGenerator(name = "uuid", strategy = "uuid2")
+  @Column(columnDefinition = "BINARY(16)")
+  private UUID id;
+
+  @ManyToOne private VeteranEntity veteran;
+
+  @NotNull private Integer flashId;
```

Comment:
I'm not familiar with this in the code, but there was talk of a String version of the flash. Did you come across a mapping of these `flashId` Integers to flash strings?

Reply:
I did not come across that mapping. @alan-amida and maybe @ianwolson might know the history on this one.

Reply:
Here's one Slack thread https://dsva.slack.com/archives/C0309PZT20Z/p1675966464451669?thread_ts=1675710347.845039&cid=C0309PZT20Z:
> Could you pls share the "short name" of the (Agent Orange Exposure Verified flash (ID: 266)) in the notification to VRO .i.e automatedClaim API request

Response:
> this is the flash name we will send. "Agent Orange Exposure Verified". We are not aware of different types of names (short/long etc.) for flashes.

This may be irrelevant but I wan't to bring it up, just in case.

Reply:
Actually now that I'm reading [this Slack thread](https://dsva.slack.com/archives/C0309PZT20Z/p1675271527188669?thread_ts=1668694256.106189&cid=C0309PZT20Z):
> We do not have any flash ids. We can send the short names of the applicable flashes as a list of strilngs.

and looking at the request object schema:
<img width="436" alt="image" src="https://user-images.githubusercontent.com/55255674/219529158-d98fde9e-b085-4c25-85b1-2c1009333327.png">

I think we should change the `flashId` into a String -- this is the safest bet.

Reply:
Agreed, we'll change it to String.